buildconfig/pygame-stubs/__init__.pyi

```diff
@@ -35,16 +35,24 @@ from pygame import fastevent as fastevent
 # Other
 from pygame import scrap as scrap
 
-from ._common import _AnyPath
+_AnyPath = Union[str, bytes, PathLike[str], PathLike[bytes]]
```

Comment:
Perhaps `from ._common import AnyPath` would be better here?

Reply:
If I do `from ._common import` in \_\_init\_\_ file than PyCharm gives me typehint for pygame._common.
![image](https://user-images.githubusercontent.com/29337110/146672308-e8ca9303-dc96-4975-afc2-2ccb6a221696.png)
But importing from _common in other files works well